app/Http/Controllers/Forum/TopicsController.php

```diff
@@ -115,6 +115,7 @@ public function pin($id)
     public function preview()
     {
         $forum = Forum::findOrFail(Request::input('forum_id'));
+        $contentOnly = Request::input('content_only') === '1';
```

Comment:
change this whole action to just return transformed body and update the main view and js accordingly.